moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -19,6 +20,32 @@ export const formatProjectActivity = (change, lookupList) => {
     };
   }
 
+  const newRecord = changeData.new;
+  const oldRecord = changeData.old;
+  let changes = [];
+  const fieldsToSkip = [
+    "updated_at",
+    "updated_by_user_id",
+    "project_name_full",
+  ];
+
+  // loop through fields to check for differences, push label onto changes Array
+  Object.keys(newRecord).forEach((field) => {
+    // typeof(null) === "object", check that field is not null before checking if object
+    if (!!newRecord[field] && typeof newRecord[field] === "object") {
+      if (!isEqual(newRecord[field], oldRecord[field])) {
+        changes.push(entryMap.fields[field]?.label);
+      }
+    } else if (
+      newRecord[field] !== oldRecord[field] &&
+      !fieldsToSkip.includes(field)
+    ) {
+      changes.push(entryMap.fields[field]?.label);
+    }
+  });
```

Comment:
this is now the third time its in the codebase, I want to pull this into its own function in a later PR.